coalib/output/ConsoleInteraction.py

```diff
@@ -625,6 +628,15 @@ def get_action_info(section, action, failed_actions):
 
     return action.name, section
 
+def provide_all_actions():
```

Comment:
Do you need this method inspite of having access to the param `actions`? 🧐

Reply:
I needed advice on that. I was unsure if i need to keep that method or not. Should i let the apply_single method display what the original author intended it to or should i change it.

Reply:
You mean modifying the doctoring?

Reply:
Do you remember the output you posted while filing for the issue. The ouput was actually coded into the print statments . But if you see choose_action() method, it does some  magic to get only the required actions and print them.

Reply:
I think the reason thoose function were added was to maybe add an extra functionality later on. I think it's better to keep it as it is. What's your opinion?